pkgs/by-name/gz/gz-cmake/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  ruby,
+  doxygen,
+  graphviz,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gz-cmake";
+  version = "4_4.1.1";
```

```suggestion
  version = "4.1.1";
```
I think that verson might just be a repetition

---

sha256-uhVl1fqoyMF/L98PYCmM6m7+52c4mWj2qlna5sz/RxE=

---

it does seem to me like the 4_4 is repetition, the version should be "4.1.1" and the tag can be `gz-cmake${lib.versions.major finalAttrs.version}_${finalAttrs.version}`